### DISCUSSION TITLE
Can I use multiple fastapi docs and click on link and open respective doc looks like below format

### DISCUSSION THREAD

**Discussion by priyankaatigre168** [fastapi/fastapi#5276] [Questions] [answered]
Services[sidebar]
							Test
							GET /test

1.Test
2.TestData
3TestPython

same like swagger UI doc with sidebar option and after clicking its opening its own doc details with respective api's

**Comment by YuriiMotov** [comment, marked as answer]
Not sure it will be easy, but you can try configuring Swagger UI.
Related docs: https://fastapi.tiangolo.com/how-to/configure-swagger-ui/
